product_docs/docs/pem/9/supported_platforms.mdx

```diff
@@ -10,12 +10,12 @@ For information about the platforms and versions supported by PEM, see [Platform
 
 ## Database compatibility
 
-The following table provides information about the PEM versions and their supported corresponding versions of PostgreSQL (PG), EDB Postgres Extended Server (PGE), EDB Postgres Advanced Server (EPAS), and EDB Postgres Distributed (PGD).
+This table provides information about the supported versions of PostgreSQL (PG), EDB Postgres Extended Server (PGE), EDB Postgres Advanced Server (EPAS), and EDB Postgres Distributed (PGD). 
```

Comment:
Add " for PEM 9.x" at the last of intro sentence and remove from the table header

Reply:
Done